<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>471</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblKeyword">
       <property name="text">
        <string>Keyword : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tbxKeyword"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnImg_1">
       <property name="text">
        <string>Git</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImg_2">
       <property name="text">
        <string>Git</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImg_3">
       <property name="text">
        <string>Git</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>471</width>
      <height>166</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="img_1">
       <property name="text">
        <string>foto 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_2">
       <property name="text">
        <string>foto 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="img_3">
       <property name="text">
        <string>foto 3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btnDownload_1">
       <property name="text">
        <string>İndir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDownload_2">
       <property name="text">
        <string>İndir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDownload_3">
       <property name="text">
        <string>İndir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
